include/Application/VggEnv.hpp

```diff
@@ -34,6 +35,7 @@ class VggEnv : public IVggEnv
 
   DarumaContainer m_darumaContainer;
 
+  UIApplication*            m_application;
```

Comment:
Why not `weak_ptr`?

Reply:
It's owned by a unique_ptr in class `AppBase` or `ContainerImpl`.